### PR TITLE
docs: Fix API in readme

### PR DESCRIPTION
**Pull request by ffflorian**
cc @malept

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,14 +34,15 @@ Options:
...
 
 ### API
 
 ```javascript
-var createDMG = require('electron-installer-dmg')
-createDMG(opts, function done (err) { })
+const createDMG = require('electron-installer-dmg')
+createDMG(opts).then(() => {
+  // ...
+})
```
Comment (malept) [inline README.md:45]: suggestion(blocking): I'd prefer an example along the lines of either:

* https://electron.github.io/electron-packager/v15.2.0/modules/electronpackager.html
* https://github.com/electron/get#usage

I think showing the `async`/`await` syntax as an example is preferable to raw Promise syntax.

**Review by malept — CHANGES_REQUESTED** [review]
I appreciate the docs update! I have a code style suggestion below.

File: README.md (hunk 1 above)
Comment (ffflorian) [reply README.md]: I updated the readme accordingly.

File: README.md [hunk 2]
```diff
@@ -77,13 +82,13 @@ How big to make the icon for the app in the DMG. [Default: `80`].
...
-Array Example:
-```
+Array example:
+```javascript
 [ { x: 448, y: 344, type: 'link', path: '/Applications'},
   { x: 192, y: 344, type: 'file', path: '/path/to/application.app'} ]
 ```
-Function Example (more flexible for getting useful options used in creating dmg):
-```
+Function example (more flexible for getting useful options used in creating DMG):
```
Comment (malept) [inline README.md:90]: Grammar nit while we're changing this line:

```suggestion
Function example (more flexible for getting useful options used in creating a DMG):
```

**Review by malept — APPROVED** [review]

**Pull request merged**